src/makeRobotAccountant.js

```diff
@@ -6,7 +6,20 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let callCount = 0;
+
+  const getSum = (num1) => {
```

Comment:
don't use explicit numbers in variable names
```suggestion
  const getSum = (firstNumber) => {
```